xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -189,7 +189,7 @@
       <Docs>
         <param name="array">To be added.</param>
         <param name="arrayIndex">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Copies the elements of the <see cref="T:System.Collections.Generic.ICollection`1" /> to an <see cref="T:System.Array" />, starting at a particular <see cref="T:System.Array" /> index.</summary>
```

Comment:
These crefs that were imported from the parent interface need to be adjusted to point to the current type. For example:
```suggestion
        <summary>Copies the elements of the <see cref="T:System.Text.RegularExpressions.CaptureCollection" /> to an <see cref="T:System.Array" />, starting at a particular <see cref="T:System.Array" /> index.</summary>
```
Makes sense?